config/src/chain.rs

```diff
@@ -97,12 +97,12 @@ impl From<Chain> for U256 {
 }
 
 impl TryFrom<Chain> for ethers_core::types::Chain {
-    type Error = ParseChainError;
+    type Error = String;
```

Comment:
https://github.com/foundry-rs/foundry/pull/3937/commits/322ba09a4cee500e77c49ef488ecb7037c909b53

```suggestion
    type Error = <ethers_core::types::Chain as TryFrom<u64>>::Error;
```